backend/src/apiserver/config/sample_config.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "name":"[Sample] ML - XGBoost - Training with Confusion Matrix",
-    "description":"A trainer that does end-to-end distributed training for XGBoost models. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost-spark",
+    "description":"A trainer that does end-to-end distributed training for XGBoost models. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost-spark. Note that this sample requires: 1) Enable the DataProc API: https://cloud.google.com/endpoints/docs/openapi/enable-api for the given GCP project.; 2) Follow GCP Service Account: https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials to create a service account with the Dataproc permissions and a corresponding kubernetes secret.",
```

Comment:
These long description are not even viewable on KFP UI. 
@Bobgy, is it possible to implement auto-popup to show the full description when user moves mouse onto description field?
@gaoning777, what we also need a hyperlink to point to sample README file. The current UI experience of Description is broken

Reply:
PR: https://github.com/kubeflow/pipelines/pull/2376

Reply:
> These long description are not even viewable on KFP UI.
AFAIK, the description in the sample.config(only for preloaded samples) is loaded by the backend and rendered in the UI. 
> @Bobgy, is it possible to implement auto-popup to show the full description when user moves mouse onto description field?
> @gaoning777, what we also need a hyperlink to point to sample README file. The current UI experience of Description is broken
By "broken", do you mean that the Pipeline description in the SDK is not displayed on the UI? If so, I think it has never been supported and I created an issue to track: https://github.com/kubeflow/pipelines/issues/2379

Reply:
@Bobgy, if the description is long enough, there is a chance that the link is not visible on the screen, hence user can't click on it. Last I checked, user can't scroll the description to get a full content. This is what I meant by "broken".  Love your demo though, very self-explanatory!